docs/plugins/google-cloud.md

```diff
@@ -1,72 +1,116 @@
-[Google Cloud's operations suite](https://cloud.google.com/products/operations) which powers the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457).
-
-> Note: Logging is facilitated by [Winston](https://github.com/winstonjs/winston) in favor of the [OpenTelemetry](https://opentelemetry.io/) logging APIs. Export of logs is done via a dedicated Winston Google Cloud exporter.
+The Google Cloud plugin exports Firebase Genkit telemetry and logging data to
+the 
+[Cloud Observability](https://cloud.google.com/products/operations)
+suite, which powers the 
+[Firebase AI Monitoring](https://forms.gle/Lp5S1NxbZUXsWc457) dashboard.
 
 ## Installation
 
 ```posix-terminal
 npm i --save @genkit-ai/google-cloud
 ```
 
-If you want to locally run flows that use this plugin, you also need the
-[Google Cloud CLI tool](https://cloud.google.com/sdk/docs/install) installed.
+When running Genkit code locally that includes this plugin, you will also need 
+the [Google Cloud CLI tool](https://cloud.google.com/sdk/docs/install)
+installed.
 
 ## Set up a Google Cloud account
 
-This plugin requires a Google Cloud account ([sign up](https://cloud.google.com/gcp) if you don't already have one) and a Google Cloud project.
+This plugin requires a Google Cloud account/project. All Firebase projects
+include one by default ([GCP Console](https://console.cloud.google.com)), 
+or you can sign up at https://cloud.google.com.
 
-Prior to adding the plugin, make sure that the following APIs are enabled for your project:
+Prior to adding the plugin, make sure that the following APIs are enabled for 
+your GCP project:
 
 - [Cloud Logging API](https://console.cloud.google.com/apis/library/logging.googleapis.com)
 - [Cloud Trace API](https://console.cloud.google.com/apis/library/cloudtrace.googleapis.com)
 - [Cloud Monitoring API](https://console.cloud.google.com/apis/library/monitoring.googleapis.com)
 
-These APIs should be listed in the [API dashboard](https://console.cloud.google.com/apis/dashboard) for your project.
+These APIs should be listed in the
+[API dashboard](https://console.cloud.google.com/apis/dashboard) for your 
+project.
 
-Click [here](https://support.google.com/googleapi/answer/6158841) to learn more about enabling and disabling APIs.
+Click [here](https://support.google.com/googleapi/answer/6158841) to learn more 
+about enabling and disabling APIs.
 
 ## Genkit configuration
 
-To enable exporting to Google Cloud Tracing, Logging, and Monitoring, simply call `enableGoogleCloudTelemetry()`:
+To enable Cloud Tracing, Logging, and Monitoring (metrics), simply call 
+`enableGoogleCloudTelemetry()`:
 
 ```ts
 import { enableGoogleCloudTelemetry } from '@genkit-ai/google-cloud';
 
 enableGoogleCloudTelemetry();
 ```
 
-When running in production, your telemetry gets automatically exported.
+When running in production, telemetry will automatically be exported.
```

Comment:
super nit: "will be automatically exported" or "will be exported automatically" may sound a little smoother

Reply:
I am also a grammer nerd, ty